packages/discord.js/src/client/actions/MessagePollVoteAdd.js

```diff
@@ -13,9 +16,26 @@ class MessagePollVoteAddAction extends Action {
 
     const { poll } = message;
 
-    const answer = poll?.answers.get(data.answer_id);
-    if (!answer) return false;
+    const includePollPartial = this.client.options.partials.includes(Partials.Poll);
+    if (message.partial && !includePollPartial) return false;
 
+    if (!poll && includePollPartial) {
+      message.poll = new Poll(this.client, { ...data, partial: true }, message, channel);
```

Comment:
`partial: true` appears to be unused.

Reply:
This *should* be fixed now, added logic in `_patch()` to handle `partial` as a property in `data`